code/datums/ai/robot_customer/robot_customer_behaviors.dm

```diff
@@ -29,10 +29,13 @@
 		controller.blackboard[BB_CUSTOMER_MY_SEAT] = found_seat
 		attending_venue.linked_seats[found_seat] = customer_pawn
 		finish_action(controller, TRUE)
-	else
+		return
+
+	if(!controller.blackboard[BB_CUSTOMER_SAID_CANT_FIND_SEAT_LINE] || DT_PROB(0.1, delta_time))
```

Comment:
dont use dt_prob for this, deltatime is innaccurate in these actions and it's deprecated. also lol at 0.1 chance, that's ridicolously miniscule. how is the solution to something like this making it literaly happen once every 15 minutes?

Reply:
delta_time implemented more cleanly. It should no longer be deprecated.